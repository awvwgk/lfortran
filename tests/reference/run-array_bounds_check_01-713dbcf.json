{
    "basename": "run-array_bounds_check_01-713dbcf",
    "cmd": "lfortran --no-color {infile}",
    "infile": "tests/errors/array_bounds_check_01.f90",
    "infile_hash": "e8570ec720776191802ff5d4d5cd6944a6b2e8730e5bf1bd8458dd83",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "run-array_bounds_check_01-713dbcf.stderr",
    "stderr_hash": "f72e643b4e6e9587ef54c16e7dd6b09cb3e0063a43ce3d1364bc9e7a",
    "returncode": 1
}