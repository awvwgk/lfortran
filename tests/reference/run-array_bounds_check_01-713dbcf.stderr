At 5:5 of file tests/errors/array_bounds_check_01.f90
Runtime Error: Size mismatch in assignment to 'x'

LHS size is 10 and RHS size is 3

Use '--realloc-lhs-arrays' option to reallocate LHS automatically.
